gcp/helmfile.yaml

```diff
@@ -17,6 +17,11 @@ environments:
     values:
       - workload_identity_project_id: "%GCLOUD_PROJECT%"
         namespace: otel-demo
+        otlp_endpoint: ""
+        auth_extension_project_id: ""
+        auth_extension_quota_project_id: ""
```

Comment:
Why are you specifying these here with empty strings? If you're trying to have them disabled by default, you can just use `getOrNil` in the template and you don't need to specify them at all in the `values` section.

Reply:
sgtm, I wasn't sure if they needed to be defined or something